kolibri/plugins/management/assets/src/vue/data-page/index.vue

```diff
@@ -53,6 +48,28 @@
   const coreApp = require('kolibri');
 
   module.exports = {
+    $trNameSpace: 'manageData',
+    $trs: {
+      // Headings/subHeadings
+      pageHeading: 'Export Usage Data',
+      pageSubHeading: 'Download CSV (comma-separated value) files containing information about' +
```

Comment:
The linter doesn't let us use `/` to escape strings because it's an ES5 only feature, so I resorted to concatenation. Can anyone foresee problems with this?